sdks/go/pkg/beam/io/textio/textio.go

```diff
@@ -108,36 +106,30 @@ func expandFn(ctx context.Context, glob string, emit func(string)) error {
 	return nil
 }
 
-// sizeFn pairs a filename with the size of that file in bytes.
-// TODO(https://github.com/apache/beam/issues/20607): Once CreateInitialRestriction supports Context params and
-// error return values, this can be done in readSdfFn.CreateInitialRestriction.
-func sizeFn(ctx context.Context, filename string) (string, int64, error) {
+// readFn reads individual lines from a text file, given a filename and a
+// size in bytes for that file. Implemented as an SDF to allow splitting
```

Comment:
Technically it no longer relies on an input size in bytes. We can to remove that part of the comment now.

Reply:
Good catch, thanks!